skyVegasTests/stakeUpdateGameClient.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { test, type Page } from '@playwright/test';
+import {
+    launchGame,
+    login,
+    validateConsoleMessages,
+    startEventListener,
+    readGames,
+} from './utilities';
+import { ExpectedMessage } from '../types/expectedMessage';
+
+let page: Page;
+readGames().forEach((game) => {
+    const consoleMessages: string[] = [];
+
+    test.describe(`Testing with text: ${game}`, async () => {
+        test.beforeAll(async ({ browser }) => {
+            page = await browser.newPage();
+            await login(page);
+        });
+
+        test.afterAll(async () => {
+            await page.close();
+        });
+
+        test('stakeUpdate', async () => {
+            startEventListener(page, consoleMessages);
+            await launchGame(page, game, consoleMessages);
+
+            await page.mouse.click(300, 300, { delay: 10 });
```

Comment:
We can refactor this into a helper function to get past the loading screen. I have noticed that some games from pragmatic don't have a loading screen, but some do. You know you have gone past the loading screen when this message is displayed it seems `"playModeUpdate", payload: "{"playMode":"normal"}`.

 So we know that our game doesn't have a loading screen if this message already exists before making the first spin. If it doesn't exist then it means we need to get past the loading screen first before making a spin.

Reply:
Discussed this, won't do anymore.